tardis/plasma/base.py

```diff
@@ -258,10 +260,35 @@ def remove_hidden_properties(self, print_graph):
                 print_graph.remove_node(str(item.name))
         return print_graph
 
-class StandardPlasma(BasePlasma):
+    def to_hdf(self, path_or_buf, path='', close_hdf=True, filter=None):
```

Comment:
take out the filter. I think it's some complicated beast that we can just do with a list.